ci_framework/roles/edpm_prepare/tasks/main.yml

```diff
@@ -101,6 +101,19 @@
           --namespace={{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }}
           --for=jsonpath='{.status.phase}'=Complete --timeout=20m
 
+- name: Patch CSV for ansible runner image
+  when:
+    - not cifmw_edpm_prepare_dry_run
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.command:
+    cmd: >-
+      oc patch csv -n openstack-operators openstack-ansibleee-operator.v0.0.1
+      --type='json' -p='[{"op":"replace",
+      "path":"/spec/install/spec/deployments/0/spec/template/spec/containers/1/env/0",
+      "value": {"name": "ANSIBLEEE_IMAGE_URL_DEFAULT", "value": ${ANSIBLEEE_IMAGE_URL_DEFAULT}}}]'
```

Comment:
@rabi I don't think I can update your PR as it was sent from your local fork. I was thinking of updating with something like below using ansible role var here:

~~~
-      "value": {"name": "ANSIBLEEE_IMAGE_URL_DEFAULT", "value": ${ANSIBLEEE_IMAGE_URL_DEFAULT}}}]'
+      "value": {"name": "ANSIBLEEE_IMAGE_URL_DEFAULT", "value": {{cifmw_edpm_prepare_ansible_runner_image}}}}]'
~~~


and specify in defaults/main.yaml

~~~
+cifmw_edpm_prepare_ansible_runner_image: "http://quay.io/openstack-k8s-operators/openstack-ansibleee-runner:latest"
~~~

and maybe we can only patch based on condition
~~~
when: cifmw_edpm_prepare_ansible_runner_image != "http://quay.io/openstack-k8s-operators/openstack-ansibleee-runner:latest"
~~~